File: yangxj96-bean/src/main/java/io/github/yangxj96/bean/system/Configure.java
package io.github.yangxj96.bean.system;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import io.github.yangxj96.common.base.BasicEntity;
import lombok.*;
import lombok.experimental.SuperBuilder;

import java.io.Serializable;

/**
    * 系统配置表
    */
@Data
@EqualsAndHashCode(callSuper=true)
@SuperBuilder
@AllArgsConstructor
@NoArgsConstructor
@TableName(value = "db_system.t_configure")
public class Configure extends BasicEntity implements Serializable {

    /**
     * key
     */
    @TableField(value = "\"key\"")
    private String key;

    /**
     * value
     */
    @TableField(value = "\"value\"")
    private String value;
}

File: yangxj96-bean/src/main/java/io/github/yangxj96/bean/system/Configure.kt
package io.github.yangxj96.bean.system

import com.baomidou.mybatisplus.annotation.TableField
import com.baomidou.mybatisplus.annotation.TableName
import io.github.yangxj96.common.base.BasicEntity
import java.io.Serializable

/**
 * 系统配置表
 */
@TableName(value = "db_system.t_configure")
class Configure : BasicEntity(), Serializable {

    /**
     * key
     */
    @TableField(value = "\"key\"")
    var key: String? = null

    /**
     * value
     */
    @TableField(value = "\"value\"")
    var value: String? = null
}